packages/ses/src/tame-faux-data-properties.js

```diff
@@ -0,0 +1,124 @@
+import {
+  getOwnPropertyDescriptor,
+  apply,
+  defineProperty,
+  toStringTagSymbol,
+} from './commons.js';
+
+/**
+ * Exported for convenience of unit testing. Harmless, but not expected
+ * to be useful by itself.
+ *
+ * @param {any} obj
+ * @param {string|symbol} prop
+ * @param {any} expectedValue
+ * @returns {boolean}
+ */
+export const tameFauxDataProperty = (obj, prop, expectedValue) => {
+  if (obj === undefined) {
+    return false;
+  }
+  const desc = getOwnPropertyDescriptor(obj, prop);
+  if (!desc || 'value' in desc) {
+    return false;
+  }
+  const { get, set } = desc;
+  if (typeof get !== 'function') {
+    return false;
+  }
+  if (typeof set !== 'function') {
+    return false;
+  }
+  if (apply(get, obj, []) !== expectedValue) {
+    return false;
+  }
+  const testValue = 'Seems to be a setter';
```

Comment:
No change recommended here, but reasoning aloud:

Proposition: maybe this value should be non-forgeable, like `Symbol()` or `{}` so we cannot be confused by a clever implementation.

However, the clever implementation must necessarily be part of the TCB because it runs before SES. We are not looking to avoid confusion from an adversary, only accidental coïncidence.

Also, however, “Seems to be a getter” is very unlikely to collide accidentally.

Also, however, by sending the value to the setter, being non-forgeable is irrelevant: the clever implementation will have the value immediately.

Reply:
From this, I infer you're ok with the current strategy? Resolving. But if I misunderstood, please unresolve.